### DISCUSSION TITLE
MDITA doesn't support code spans

### DISCUSSION THREAD

**Discussion by jelovirt** [jelovirt/org.lwdita#109] [Ideas]
if you look at the old LwDITA committee note or the [current spec draft](https://github.com/oasis-tcs/dita-lwdita/tree/spec/Spec/components), there is actually no support for [code spans](https://spec.commonmark.org/0.30/#code-spans) with a single backtick. The LwDITA plug-in has supported this in the past.

If I actually start following the spec draft, everyone who uses `format="mdita"` and uses code spans, will no longer get `<codeph>` and will instead get `<ph>` or an error.

So a heads up, that you might want to check what MDITA supports and use Markdown instead if you want e.g. code spans.

**Comment by kirkilj** [comment]
FYI: When I run the Gradle test task, I get a failure, but it's only due to the fact that there's an extra blank line at the end of the actual results compared to the expected results.

**Comment by kirkilj** [comment]
Several flexmark extensions are commented in out in various build files. Is there a distinction between any could be converted to DITA, but weren't a priority and those which make no sense in the DITA context?

**Reply by jelovirt** [reply]
There are probably features of various Markdown flavours that could be supported with `format="markdown"`, but I haven't considered important at the time.

**Comment by kirkilj** [comment]
I'm interviewing some job candidates with senior Java dev experience this week so we can contribute to this project and help in this project as well as DITA-OT. Even though I taught Java to high-school students and perform code-reviews on our custom Oxygen Java plugin, I am not at the level where I could personally contribute, so any general dev notes on how to incorporate new capabilities into LW-DITA's Markdown to DITA conversion would be appreciated.
